src/workerd/server/server.h

```diff
@@ -112,6 +112,10 @@ class Server final: private kj::TaskSet::ErrorHandler {
   class InspectorService;
   class InspectorServiceIsolateRegistrar;
 
+  void handleReportConfigError(kj::String error) {
```

Comment:
Why not just use reportConfigError?

Reply:
`reportConfigError(...)` is private and the callsite for this is in an anonymous namespace in a utility class. I could move `reportConfigError(...)`, I suppose but this seems just as easy.

Reply:
Yeah I mean, creating a public method that exports a private method is kinda funny.

Reply:
Well, `reportConfigError` is not a private method. It's a private field member whose type happens to be a function...

```
kj::Function<void(kj::String)> reportConfigError;
```